portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,84 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+  background-color: #aaa;
+  margin: 0;
+  padding: 0;
+}
+
+#header-content {
+  background-color: #aaa;
+  display: flex;
+  height: 5%;
+  margin: 0;
+  padding: 0;
+  width: inherit;
+}
+
+#image-container {
+  display: flex;
+  flex-direction: column;
+}
+
+#cycle-button {
+  background-color: #888;
+  color: #000;
+  border: none;
+}
+
+#image-container button:hover {
+  background-color: #00f;
+}
+
+#header-selfie {
+  height:450px; 
+  width:300px;
 }
 
 #greeting-container {
-  margin-top: 20px;
+  background-image: url(images/skyline.jpg);
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+#greeting-container h1,
+ h3 {
+  color: #F00;
+}
+
+#greeting-container:hover #header-nav {
+  opacity: 1;
+  transition: none;
+}
+
+#greeting-container:not(:hover) #header-nav {
+  opacity: 0;
+  transition: opacity 3s;
+  transition-delay: 2s; 
```

Comment:
You can use this shorthand here:
```css
  transition: opacity 3s linear 2s;
```
but I would recommend a different timing function since it can help things look smoother
```css
  tansition: opacity 3s ease 2s;
```

It also helps to not mix shorthand properties (e.g. `transition`) and their specific sub-properties (i.e. `transition-delay`) in the same rule since it makes it harder to reason how these properties are overridden by other rules. For example,

```css
.header-nav {
  opacity: 0;
  transition: opacity 3s;
  transition-delay: 2s; 
}

.header-nav.fast {
  transition: opacity 1s ease 0;
}
```

Here you might expect `transition-delay` to be 0 for `.header-nav.fast` but since the properties were mixed it might actually be `2s`.

Reply:
Oh okay thank you, I was a little confused on how the transition worked earlier.

Reply:
The reason I have the transition delay is so when the page is first loaded, it waits 2 seconds before it starts to fade out. That way the nav bar is still displayed before a user hovers over the greeting div.

Reply:
Done. I was a little confused how to the transition worked earlier.